frontend/src/components/worksheets/NewRun/NewRun.jsx

```diff
@@ -162,7 +162,7 @@ class NewRun extends React.Component<{
     defaultConfig = {

```

Comment:
Remove name field placeholder, empty name will automatically be filled with run-<command> upon submission